### PR TITLE
Add cloudman, helm-docs, chart-testing

### PR DESCRIPTION
**Pull request by patsevanton**

### CODE REVIEW

File: distributions/distributions.yaml [hunk 1]
```diff
@@ -304,6 +304,19 @@ sources:
...
+    url: https://github.com/helm/chart-testing
+    list:
+      type: github-releases
+      url: https://api.github.com/repos/helm/chart-testing/releases
+    fetch:
+      url: https://github.com/helm/chart-testing/releases/download/v{{ .Version }}/chart-testing_{{ .Version }}_{{ .OS }}_{{ .Arch }}.tar.gz
+    install:
+      type: tgz
+      binaries:
+        - chart-testing
```
Comment (leucos) [inline distributions/distributions.yaml:318]: Note that the proper name here is `ct`

I fixed locally after merging

**Comment by patsevanton** [comment]
Thanks!
distributions.yaml need change in PR ?

**Comment by leucos** [comment]
> Thanks! distributions.yaml need change in PR ?

No no, I fixed it already. It is all good.

**Comment by patsevanton** [comment]
When i will new PR - what is file i need change?

**Comment by leucos** [comment]
only distributions.yaml

**Pull request merged**